### PR TITLE
Content: Aerosol Discovery

### PR DESCRIPTION
**Pull request by WeathermanTrent**
## Adding content

Working with Aerosol Optical Depth data. 

---

## Content checklist

Check the [Content Documentation](https://github.com/NASA-IMPACT/veda-config/blob/main/docs/CONTENT.md) for more details.

### Any content
- [x] Cover image with proper attribution (when relevant) is present.
- [x] Cover images do not contain any text (they're used as a background and text may not be readable).
- [x] All images are not too big/heavy. As a general rule of thumb, they should not weigh more than 500KB, or be larger than 2000px. See the [media guide](https://github.com/NASA-IMPACT/veda-config/blob/main/docs/frontmatter/media.md#media) for more info.
- [x] Content was added using the available [MDX Blocks](https://github.com/NASA-IMPACT/veda-config/blob/main/docs/MDX_BLOCKS.md).

### Datasets
- [x] [Layer description](https://github.com/NASA-IMPACT/veda-config/blob/main/docs/frontmatter/layer.md#properties) is defined.
- [x] [Layer legend](https://github.com/NASA-IMPACT/veda-config/blob/main/docs/frontmatter/layer.md#legend) is setup.
- [x] The Dataset is showing up on the map correctly

### Discoveries
- [x] [Publication date](https://github.com/NASA-IMPACT/veda-config/blob/main/docs/CONTENT.md#discoveries) value is set

### CODE REVIEW

File: datasets/aerosol-dataset.data.mdx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+id: houston-aod
+name: "Houston Aerosol Optical Depth"
+description: "The impact of Aerosol over the Houston-metro"
+media:
+  src: ::file ./smog-national-geographic-cover.jpg
+  alt: Smog over a city.
+  author:
+    name: Trent Cowan
```
[the remark this answers is not in the document]
Comment (slesaad) [reply datasets/aerosol-dataset.data.mdx:9]: the url doesn't match either

[the remark this answers is not in the document]
Comment (WeathermanTrent) [reply datasets/aerosol-dataset.data.mdx:9]: The URL seems to be working on my end. It sends me to the page where I found the smokestack.

[the remark this answers is not in the document]
Comment (slesaad) [reply datasets/aerosol-dataset.data.mdx:9]: this is for smog-national-geographic-cover.jpg
although it doesn't even match the other image

[the remark this answers is not in the document]
Comment (WeathermanTrent) [reply datasets/aerosol-dataset.data.mdx]: Ahh, I see this is the dataset. Just fixed

[the remark this answers is not in the document]
Comment (slesaad) [reply datasets/aerosol-dataset.data.mdx]: Delete all the images that you are no longer using

File: discoveries/houston-aod.discoveries.mdx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+   Using satellite data from the MODIS Terra and Aqua instruments that observe Earth at different times of the day and provide a global view 1-2 days. This satellite provides crucial information such as Aerosol thickness that aides in determing the trend in aerosol patterns to help determine the track for future pollution. 
+  </Prose>
+</Block>
+
+<Block type="full">
+  <Figure>
+    <Map
+      datasetId='aerosol-dataset'
+      layerId='houston-landcover'
```
Comment (slesaad) [inline discoveries/houston-aod.discoveries.mdx:47]: are you trying to use the `houston-landcover` or `houston-aod` layer? the `houston-landcover` layer doesn't exist in the `aerosol-dataset` dataset. It's probably in some other dataset

**Comment by slesaad** [comment]
@WeathermanTrent make sure you're following everything in the checklist

File: discoveries/houston-aod.discoveries.mdx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  </Prose>
+</Block>
+
+<Block type="full">
+  <Figure>
+    <Map
+      datasetId='houston-aod'
+      layerId='houston-aod'
+      dateTime='2001-02-01'
+      compareDateTime='2019-12-31'
```
[the remark this answers is not in the document]
Comment (WeathermanTrent) [reply discoveries/houston-aod.discoveries.mdx:49]: Fixed!

**Comment by WeathermanTrent** [comment]
This is complete. Do I close the pull request?

File: datasets/Harris_AOD_00_09_county.jpg
[the remark this answers is not in the document]
Comment (WeathermanTrent) [reply datasets/Harris_AOD_00_09_county.jpg:1]: Yes! That is my bad, I completely forgot they were there.

**Comment by j08lue** [comment]
Great to see this! 

One thing that would be great to add are the taxonomies - `sources` and `thematics`. Could you still add those, please @WeathermanTrent? The docs on that are here: https://github.com/NASA-IMPACT/veda-ui/blob/main/docs/content/TAXONOMY.md

> who is the best person for content review

Can you nominate someone who can approve this content to go public, @WeathermanTrent? Otherwise, let us post this in the VEDA Slack channel and see who is eligible, perhaps pinging @deborahUAH.

**Comment by WeathermanTrent** [comment]
> 

I can add those changes! When you say nominate someone, should I nominate @slesaad or @hanbyul-here ? Is that what you mean?

**Comment by j08lue** [comment]
No, I meant someone at NASA to can offer scientific quality assurance and, most importantly, programmatic approval (that this content is ok to publish under IMPACT's name). If in doubt, let's ask @deborahUAH.

**Comment by WeathermanTrent** [comment]
> No, I meant someone at NASA to can offer scientific quality assurance and, most importantly, programmatic approval (that this content is ok to publish under IMPACT's name). If in doubt, let's ask @deborahUAH.

Ahh, I see! Asking @deborahUAH would be best then.

**Comment by j08lue** [comment]
Review is now in progress. Sounds like there will be more iterations on the content, @hanbyul-here. I'll make sure we get a ping when all is ready.

**Comment by WeathermanTrent** [comment]
We are making a couple final touches to the introduction.

**Comment by j08lue** [comment]
@WeathermanTrent this is just to let you know that we are working on creating a tool that should make editing these MDX files a lot easier - at least give you an a lot faster feedback loop compared to the CI builds. Basically, you will see MDX changes you make render live while you make them. Sorry we did not have it ready before you had to go through all of these edits... We will keep you posted on when this tool will be available to you.

Let us know when your discovery is ready for review (by IMPACT reviewers and then someone from the dev team).

**Comment by freitagb** [comment]
@aboydnw @j08lue - we've gotten approval to publish this story. What's the next step for getting this into production?

**Comment by j08lue** [comment]
Oops, missed the ping there about the story being ready. 🎉 

@brianmfreitag, I take it the story, as it is shown here, has been reviewed and is ready for publication.

I asked @hanbyul-here for a pro-forma / technical review and then we can merge and ship it with our next push to production, which we can prepare for right away.

**Comment by freitagb** [comment]
@j08lue @hanbyul-here - yes. The data story has been reviewed and is ready for publication to the dashboard.

**Review by danielfdsilva — APPROVED** [review]

**Pull request merged**